lib/LaTeXML/Package/xkeyval.sty.ltxml

```diff
@@ -20,16 +20,17 @@ use LaTeXML::Util::Pathname qw(pathname_split);
 ###
 ### Basic \setkeys
 ###
+AssignValue('keyval.sty_loaded' => 1, 'global');    # pretend keyval loaded too.
```

Comment:
is it loading in the pdflatex flow? If so, why not `RequirePackage` instead?

Reply:
It's really just to keep `keyval` from being loaded later and overwriting `xkeyval` defs (the latter is proper superset)